docs/resources/pipeline_schedule.md

```diff
@@ -24,7 +24,7 @@ resource "buildkite_pipeline_schedule" "repo2_nightly" {
 * `commit` - (Optional, Default: `HEAD`) The commit ref to use for the build.
 * `message` - (Optional, Default: `Scheduled build`) The message to use for the build.
 * `env` - (Optional) A map of environment variables to use for the build.
-* `enabled` - (Optional, Default: `true`) Whether the schedule should run.
+* `enabled` - (Optional, Default: `true`) Whether the schedule should run. 
```

Comment:
We have a .editorconfig file in this repo that should stop these empty spaces at the end of lines. You should install the plugin for your IDE so we don't get these diffs